lib/menu.rb

```diff
@@ -0,0 +1,11 @@
+class Menu
+  attr_reader :menu
+
+  def initialize(menu)
+    @menu = menu
+  end
+  
+  def show_menu 
```

Comment:
Could iterate through menu for it to display each item in a clean manner. Also makes more use of the method